lib/instrumentation/director.js

```diff
@@ -14,36 +16,47 @@ module.exports = function initialize(agent, director, moduleName, shim) {
 
   const methods = ['on', 'route']
   const proto = director.Router.prototype
-  shim.wrapMiddlewareMounter(proto, methods, {
-    route: shim.SECOND,
-    wrapper: function wrapMiddleware(shim, middleware, name, path) {
-      return shim.recordMiddleware(middleware, {
-        route: path,
-        req: function getReq() {
-          return this.req
-        },
-        params: function getParams() {
-          return this.params
-        },
-        next: shim.LAST
-      })
-    }
-  })
+  shim.wrapMiddlewareMounter(
+    proto,
+    methods,
+    new MiddlewareMounterSpec({
+      // do I use MiddlewareMounterSpec here
```

Comment:
@bizob2828 I use the MiddlewareMounterSpec here, correct?

Reply:
yea this is correct

Reply:
I would remove that comment